modules/administration/pages/tshoot-rpctimeout.adoc

```diff
@@ -54,3 +61,4 @@ timeout =`number`
 ====
 If you limit RPC timeouts to less than `180` seconds, you risk aborting perfectly normal operations.
 ====
+
```

Comment:
I don't think you need all these `+` signs ... ?

Reply:
not sure why they were added. @jcayouette ?